compose/wordpress.md

```diff
@@ -60,7 +60,12 @@ Compose to set up and run WordPress. Before starting, you'll need to have
         db_data:
     ```
 
-    **NOTE**: The docker volume `db_data` will persist any updates made by wordpress to the database. [Learn more about docker volumes](../engine/tutorials/dockervolumes.md)
+   > **Notes:**
+   >
+   * The docker volume `db_data` will persist any updates made by wordpress to the database. [Learn more about docker volumes](../engine/tutorials/dockervolumes.md)
```

Comment:
Can you make the link absolute? You only have to remove the `..` from the front.

Also I think the style guide says that we put the colon outside the bold, for note titles. Like `**Note**:`